README.md

```diff
@@ -4,13 +4,23 @@
 
 `atm-cli` is a command line tool for generating and working with MIDI files. It was purpose-built for
 All the Music, LLC to assist in its mission to enable musicians to make all of their music
-without the fear of frivolous copyright lawsuits. All code is released into the public domain
-via the [Creative Commons Attribute 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
+without the fear of frivolous copyright lawsuits. All code is freely available under the
```

Comment:
Link

Reply:
> Link

I am confused by this comment. The link is in the next line.